examples/root_dirs/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+
+ts_project(
+    name = "lib",
+    srcs = ["src/lib.ts"],
+    out_dir = "dist",
+    declaration = True,
+    tsconfig = {},
+)
+
+ts_project(
+    name = "specs",
+    srcs = ["src/spec.ts"],
+    deps = [":lib"],
+    out_dir = "test",
+    tsconfig = {
+      "compilerOptions": {
+            "rootDirs": [
```

Comment:
I could add an assertion that this file was outputted to the `test` directory?

This is testing that the `rootDirs: ["dist"]` consumes the `out_dir = "dist",` of the other target.

It was more interesting adding `rootDir` as well but I couldn't even get that working with plain tsc.

Reply:
I see the assertion now.

rootDirs is confusingly not the plural of rootDir, the two are unrelated

Reply:
Yeah I know they are unrelated.

I was trying to...
* compile the main src into a `_dist` dir
* compile specs into a `_tests` dir (which depends on the output in `_dist` so uses `rootDirs`).
* strip the `src` dir off at the same time via `rootDir` (because angular libraries do this)

I couldn't get this combo working even outside bazel so this example drops the `rootDir` part, the bazel example is dropping the `outDir` part.